static/digital-agency/index.html

```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Digital Agency</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:wght@700;800&family=Poppins:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body class="font-poppins">
+    <header class="bg-[#F6F7F9]">
+      <div
+        class="relative flex items-center justify-between px-8 py-4 lg:mx-auto lg:max-w-6xl lg:py-6"
+      >
+        <a
+          class="font-nunito font-extrabold text-[#1A0C06] sm:text-lg md:text-xl lg:text-2xl"
+          href="/"
+          >Agency</a
+        >
+        <button class="relative z-30 sm:hidden" id="toggle-menu-btn">
+          <img
+            class="h-6 w-6"
+            src="./assets/hamburger.svg"
+            alt="hamburger icon"
+          />
+          <img
+            class="hidden h-6 w-6"
+            src="./assets/close-btn.svg"
+            alt="close icon"
+          />
+        </button>
+        <nav
+          class="absolute top-1 right-4 z-20 hidden w-56 space-y-4 rounded-md bg-white px-8 py-8 shadow-md sm:static sm:block sm:w-auto sm:space-y-0 sm:space-x-5 sm:bg-transparent sm:p-0 sm:text-xs sm:text-[#616161] sm:shadow-none lg:text-base"
+          id="nav-menu"
+        >
+          <a class="block sm:inline-block" href="#">Home</a>
+          <a class="block sm:inline-block" href="#">Our Services</a>
+          <a class="block sm:inline-block" href="#">Projects</a>
+          <a class="block sm:inline-block" href="#">Blog</a>
+          <a class="block sm:inline-block" href="#">About</a>
+          <a
+            class="bg-primary block rounded px-4 py-1 text-center text-white hover:bg-blue-600 sm:inline-block"
+            href="#"
+            >Contact</a
+          >
+        </nav>
+        <div
+          class="fixed inset-0 z-10 hidden bg-gray-500/50 sm:hidden"
+          id="overlay"
+        ></div>
+      </div>
+    </header>
+    <main>
+      <section class="bg-[#F6F7F9] py-6 px-10 sm:px-16 sm:pt-12 sm:pb-8">
+        <div class="lg:mx-auto lg:flex lg:max-w-6xl lg:flex-col">
```

Comment:
Since the container has a title, description, and an image an `<article>...</article>` would have been a better fit here.